modules/pubxaiAnalyticsAdapter.md

```diff
@@ -0,0 +1,20 @@
+# Overview
+Module Name: PubX.io Analytics Adapter
+Module Type: Analytics Adapter
+Maintainer: phaneendra@pubx.ai
+
+# Description
+
+Analytics adapter for prebid provided by Pubx.ai. Contact alex@pubx.ai for information.
+
+# Test Parameters
+
+```
+{
+  provider: 'pubxai',
+    options : {
+      pubxId: 'xxx'   //ID provided by Pubx.ai
```

Comment:
I don't see `pubxId` being used in code? 
also, missing `,`

Reply:
`pubxId` is not used in the code directly but it will be sent to the server in the `send` function. I will fix all other issues mentioned here. Thanks for your inputs.